cmd/thyme/main.go

```diff
@@ -3,36 +3,61 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"log"
 	"os"
 	"runtime"
 
-	"github.com/jessevdk/go-flags"
+	"github.com/kovetskiy/godocs"
```

Comment:
I agree, @seletskiy is there a compelling reason to use the fork here or would you mind switching to using `github.com/docopt/docopt.go`? Thanks so much for submitting this. The help page looks great and I love the ASCII art.